### DISCUSSION TITLE
Horizontal x-collapse?

### DISCUSSION THREAD

**Discussion by acalvino4** [alpinejs/alpine#2809] [3. Feature ideas]
Is it possible to use the x-collapse directive for a horizontal show/hide scenario? If not, consider this a feature request! (unless maybe it's not even needed in that case...)

**Comment by grey4owl** [comment]
I created a PR(#3949) that enables this with a modifier `.horizontal`, default is `.vertical` but you don't have to specify it.

`x.collapse.duration.1000ms || x-collapse.vertical.duration.1000ms`  collapse vertically on height of the element
`x.collapse.horizontal.duration.1000ms` collapse horizontally on width of the element
